docs/topics/keyvault-secrets.md

```diff
@@ -0,0 +1,110 @@
+# Use Key Vault as the Source of Cluster Configuration Secrets
+
+## Overview
+
+AKS Engine enables you to source the following cluster configuration from Microsoft Azure KeyVault.
+
+For official Microsoft Key Vault documentation go [here](https://docs.microsoft.com/en-us/azure/key-vault/basic-concepts).
+
+In order to use Key Vault secrets as the source of cluster configuration secrets, you pass in a reference to the secret URI in your api model:

```

Comment:
I wouldn't repeat "secrets": just say "as the source of cluster configuration,".

Reply:
```suggestion
In order to use Key Vault as the source of cluster configuration secrets, you pass in a reference to the secret URI in your api model:
```